python-sdk/docs/astro/sql/operators/export.rst

```diff
@@ -1,16 +1,16 @@
-.. _export_table_to_file:
+.. _export_to_file:
 
 ==================================================================================
-:py:mod:`export_table_to_file operator <astro.sql.operators.export_table_to_file>`
+:py:mod:`export_to_file operator <astro.sql.operators.export_to_file>`
 ==================================================================================
 
-.. _export_table_to_file_operator:
+.. _export_to_file_operator:
 
-When to use the ``export_table_to_file`` operator
+When to use the ``export_to_file`` operator
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-The ``export_table_to_file`` operator allows you to write SQL tables to CSV or parquet files and store them locally, on S3, or on GCS. The ``export_table_to_file`` function can export data from :ref:`supported_databases` or a Pandas dataframe.
```

Comment:
@pankajastro, what do you think if we deprecated `export_table_to_file` before removing it? 

Very few users would be using it since it has been available between 1.3.1-1.3.3 (from 14 December to 4 January), but that may be a less abrupt way of making this change.

We're planning to remove it by 1.5, so we have a deadline to remove this - and add a Github ticket associated with the 1.5 milestones, so we don't forget it.

Reply:
@utkarsharma2 @sunank200 @phanikumv @rajaths010494 @kaxil WDYT?

Reply:
I like that suggestion from tatiana. 👍

Reply:
deprecated and created a ticket to remove it in 1.5.0 https://github.com/astronomer/astro-sdk/issues/1521